dist/obsapisetup

```diff
@@ -10,9 +10,9 @@
 ### BEGIN INIT INFO
 # Provides:          obsapisetup
 # X-Start-Before:    apache2
-# Should-Start:      obsstoragesetup obssrcserver
+# Should-Start:      obsstoragesetup
 # Should-Stop:       $none
-# Required-Start:    mysql
+# Required-Start:    mysql obssrcserver obsrepserver
```

Comment:
src and rep server may run on a different host. Okay, not the standard situation on the appliance, but "Should-Start" would be the better place.